__mocks__/@/components/Tables/SettingsModal.js

```
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// Minimal mock for SettingsModal to avoid element type errors in tests
+const SettingsModal = ({
+  isOpen,
+  onClose,
+  onSave,
+  onReset,
+  headers,
+  visibleColumns,
```

ESLint

[error] 'visibleColumns' is defined but never used.

(@typescript-eslint/no-unused-vars)